src/lucky_avram/ext/lucky/input_helpers.cr

```diff
@@ -80,7 +81,7 @@ module Lucky::InputHelpers
   def checkbox(field : Avram::PermittedAttribute(Bool), attrs : Array(Symbol), **html_options) : Nil
     unchecked_value = "false"
     if field.value
-      html_options = merge_options(html_options, {"checked" => "true"})
+      attrs = attrs | [:checked]
```

Comment:
Can't use `<<` here because arguments are passed by reference. Which means that when we pass `EMPTY_BOOLEAN_ATTRIBUTES`, it would pass that to other methods causing ALL radios/checkboxes to be marked as checked